src/components/main/main.jsx

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 export const Main = (props) => {
-  // eslint-disable-next-line react/prop-types
-  const {rentProposals} = props;
+  const {proposalsCount, proposalsDescription} = props;
 
-  return <h1>Rent Proposals: {rentProposals}</h1>;
+  return <div>
+    <h1>Rent Proposals: {proposalsCount}</h1>
+
+    {proposalsDescription.map((i, key) => {
```

Comment:
```suggestion
    {proposalsDescription.map((item, index) => {
```
Такое именование более естественное и лучше отражает суть аргументов

Reply:
Исправила (аргумент index больше не используется в функции в качестве key, поэтому не стала его добавлять)